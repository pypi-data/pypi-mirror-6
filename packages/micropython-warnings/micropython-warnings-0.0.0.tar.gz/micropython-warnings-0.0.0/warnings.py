# MicroPython doesn't support warnings
