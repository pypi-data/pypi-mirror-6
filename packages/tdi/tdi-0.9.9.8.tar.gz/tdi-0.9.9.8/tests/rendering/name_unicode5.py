#!/usr/bin/env python
import warnings as _warnings
_warnings.resetwarnings()
_warnings.filterwarnings('error')

from tdi import html

template = html.from_string("""
<node>
    <xnode tdi="foo"></xnode>
</node>
""".lstrip())

class Model(object):
    def render_foo(self, node):
        try:
            node[u'b\xe9lah'] = u'bl\xf6d'
        except UnicodeError:
            node[u'blah'] = u'bl\xf6d'

template.render(Model())
