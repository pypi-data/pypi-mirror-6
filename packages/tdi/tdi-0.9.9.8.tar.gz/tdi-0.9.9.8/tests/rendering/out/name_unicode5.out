<node>
    <xnode blah="bl&#246;d"></xnode>
</node>
